content/doc/book/pipeline/getting-started.adoc

```diff
@@ -22,106 +23,198 @@ This section introduces some of the key concepts to Jenkins Pipeline and
+Both Declarative and Scripted Pipeline is written in
+link:http://groovy-lang.org/[Groovy] syntax. Relevant components of
+link:http://groovy-lang.org/semantics.html[Groovy syntax] will be introduced as
+required throughout this documentation, so while an understanding of Groovy is
+helpful, it is not required to work with Pipeline.
 
-A basic Pipeline can be created in either of the following ways:
+A Pipeline can be created in one of the following ways:
 
-* By entering a script directly in the Jenkins web UI.
-* By creating a `Jenkinsfile` which can be checked into a project's source
-  control repository.
+* <<#through-blue-ocean,Through Blue Ocean>> - after setting up a Pipeline
+  project in Blue Ocean, the Blue Ocean UI helps you write your Pipeline's
+  `Jenkinsfile` and commit it to source control.
+* <<#through-the-classic-ui,Through the classic UI>> - you can enter a basic
+  Pipeline directly in Jenkins through the classic UI.
+* <<#defining-a-pipeline-in-scm,In SCM>> - you can write a `Jenkinsfile`
+  manually, which you can commit to your project's source control repository.
 
 The syntax for defining a Pipeline with either approach is the same, but while
-Jenkins supports entering Pipeline directly into the web UI, it's
+Jenkins supports entering Pipeline directly into the classic UI, it is
 generally considered best practice to define the Pipeline in a `Jenkinsfile`
 which Jenkins will then load directly from source control.
-footnoteref:[scm, https://en.wikipedia.org/wiki/Source_control_management]
-
-
-=== Defining a Pipeline in the Web UI
-
-To create a basic Pipeline in the Jenkins web UI, follow these steps:
-
-* Click *New Item* on Jenkins home page.
-
-image::pipeline/new-item-selection.png["Click *New Item* on the Jenkins home page", role=center]
-
-* Enter a name for your Pipeline, select *Pipeline* and click *OK*.
-
-[CAUTION]
-====
-Jenkins uses the name of the Pipeline to create directories on disk. Pipeline
-names which include spaces may uncover bugs in scripts which do not expect
-paths to contain spaces.
-====
-
-image::pipeline/new-item-creation.png["Enter a name, select *Pipeline*, and click *OK*", role=center]
-
-
-
-* In the *Script* text area, enter a Pipeline and click *Save*.
-
-image::pipeline/hello-world-script.png["In the *Script* text area, enter a Pipeline and click Save", role=center]
-
-* Click *Build Now* to run the Pipeline.
-
-image::pipeline/build-now.png["Click *Build Now* to run the Pipeline", role=center]
-
-
-* Click *#1* under "Build History" and then click *Console Output* to see the
-  full output from the Pipeline.
-
-image::pipeline/hello-world-console-output.png["Click *Console Output* for the Pipeline", role=center]
-
-The example above shows a successful run of a basic Pipeline created in the Jenkins
-web UI, using two steps.
-
+footnoteref:[scm,link:https://en.wikipedia.org/wiki/Source_control_management[Source control management]]
```

Comment:
Since https://en.wikipedia.org/wiki/Version_control is the destination, should the link point there, rather than to a location which immediately redirects?  The words "source control management" should remain, since they are expanding the acronym SCM, but the URL will be https://en.wikipedia.org/wiki/Version_control on Wikipedia.

Reply:
Good points again @MarkEWaite - I've updated the URLs but have kept the original wording wording.

On a related matter - and @StackScribe might have thoughts about this too ... This Wikipedia entry states that the article lacks sufficient citations. When I cite work in my documentation, I usually attempt to link through to the original and most authoritative online source I can find (which usually means not linking directly to a Wikipedia article but instead, perhaps linking through to one of its references - if an appropriate source can be found).

I guess Wikipedia articles are okay to cite if you're just citing a definition. However, I feel a little discomfort if I'm citing a Wikipedia article that indicates problems at its top.

What are your thoughts about this? Do you think something like this warrants further discussion on the Jenkins Documentation Google Group?